### PR TITLE
Platform 5181

### PR DESCRIPTION
**Pull request by abador**
Fix some problems with wikis without db

### CODE REVIEW

File: extensions/wikia/WikiFactory/Close/maintenance.php [hunk 1]
```diff
@@ -202,87 +202,103 @@ public function execute() {
 			}
 			if ( $row->city_flags & WikiFactory::FLAG_DELETE_DB_IMAGES ||
 				 $row->city_flags & WikiFactory::FLAG_FREE_WIKI_URL ) {
+				$where["city_id"] = $cityid;
```
Comment (themech) [inline extensions/wikia/WikiFactory/Close/maintenance.php:206]: Please add a comment here, maybe with the ticket number. The `$where` array is created way up in the code and it is almost impossible for the reader to understand what we're checking here.

File: extensions/wikia/WikiFactory/Close/verifyWIthEmptyDB.php [hunk 2]
```diff
@@ -0,0 +1,111 @@
+<?php
```
Comment (themech) [inline extensions/wikia/WikiFactory/Close/verifyWIthEmptyDB.php]: the name of the file has an uppercase i, it should be `verifyWithEmptyDB.php ` not `verifyWIthEmptyDB.php `

File: extensions/wikia/WikiFactory/Close/verifyWIthEmptyDB.php [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+		$this->addOption( 'first', 'Run only once for first wiki in queue' );
+		$this->addOption( 'from', 'Starting offset for wiki to be closed' );
+		$this->addOption( 'dry-run', 'List wikis that will be removed and quit' );
+		$this->addOption( 'limit', 'Limit how many wikis will be processed', false, true );
+		$this->addOption( 'sleep', 'How long to wait before processing the next wiki', false, true );
+		$this->addOption( 'cluster', 'Run for a given cluster only', false, true );
+	}
+
+	/**
+	 * 1. go through all wikis which are marked for closing and check which one
```
Comment (themech) [inline extensions/wikia/WikiFactory/Close/verifyWIthEmptyDB.php]: I think this comment should be moved up, as it nicely explains what this maintenance class does

**Review by themech — CHANGES_REQUESTED** [review]

File: extensions/wikia/WikiFactory/Close/verifyWIthEmptyDB.php (hunk 3 above)
Comment (t-tomalak) [reply extensions/wikia/WikiFactory/Close/verifyWIthEmptyDB.php]: Unfortunately I think that this comment is not about this maintenance script, it's probably copy/pasta from Close/maintennance scirpt

Comment (themech) [reply extensions/wikia/WikiFactory/Close/verifyWIthEmptyDB.php]: Oh, that's even worse :) Then we should remove it and add a proper comment explaining the purpose and usage of this script at the top of this file.

File: extensions/wikia/WikiFactory/Close/maintenance.php (hunk 1 above)
Comment (abador) [reply extensions/wikia/WikiFactory/Close/maintenance.php:206]: It's added, but you'll see it in file explorer

**Review by themech — APPROVED** [review]

**Pull request merged**